packages/lib/src/spatial-navigation/components/virtualizedList/helpers/getRange.ts

```diff
@@ -52,17 +67,61 @@ const getRangeWithoutFloatHandling = ({
    */
   if (rawStartIndex < 0) {
     const finalEndIndex = numberOfRenderedItems - 1;
+
     return { start: 0, end: positiveValueOrZero(Math.min(finalEndIndex, lastDataIndex)) };
   }
 
   if (rawEndIndex > data.length - 1) {
     const finalStartIndex = lastDataIndex - numberOfRenderedItems + 1;
+
     return { start: positiveValueOrZero(finalStartIndex), end: positiveValueOrZero(lastDataIndex) };
   }
 
   return { start: rawStartIndex, end: rawEndIndex };
 };
 
+const getRawStartAndEndIndexes = ({
```

Comment:
could we add a little TSDoc to explain what is the raw start and end indexes? that might help a little bit 😁 
(nothing too fancy, just a couple sentences to explain what we mean by "raw")